dt-posts/dt-posts-endpoints.php

```diff
@@ -375,6 +375,26 @@ public function add_api_routes() {
                 ]
             ]
         );
+
+        //Request Record Access
+        register_rest_route(
+            $this->namespace, '/(?P<post_type>\w+)/(?P<id>\d+)/request_record_access', [
+                [
+                    "methods"  => "POST",
+                    "callback" => [ $this, 'request_record_access' ],
+                    "args"     => [
+                        "user_id"   => [
```

Comment:
Hey, this param is not used any more.

Reply:
ah, and as in https://github.com/DiscipleTools/disciple-tools-theme/pull/1287
We need to add the `permission_callback` parameter